docker-compose.yml

```diff
@@ -49,6 +49,7 @@ services:
       - AWS_SESSION_TOKEN=${AWS_SESSION_TOKEN}
       - MODULE_NAME=process_file.app
       - KAFKA_BOOTSTRAP_SERVERS=kafka:9093
+      - C_FORCE_ROOT=true
```

Comment:
why do we need this?

Reply:
when using pickle serializer in celery, and running celery as root in the docker container, this needs to be set. Going to check to make sure though that our real deployments arent running celery as root with plate team

Reply:
gotcha. Mind adding a comment to this effect, so we can know if/when we can remove it?